codec_specs.md

```diff
@@ -46,7 +46,7 @@ The private data contains the VCM structure BITMAPINFOHEADER including the extra
 
 | V_THEORA | Theora | The private data contains the first three Theora packets in order. The lengths of the packets precedes them. The actual layout is: 
 
-* Byte 1: number of distinct packets '`#p`' minus one inside the CodecPrivate block. This should be '2' for current Theora headers. 
+* Byte 1: number of distinct packets '`#p`' minus one inside the CodecPrivate block. This SHOULD be '2' for current Theora headers. 
```

Comment:
As of 2016-07-08 it's a MUST, but it depends on the interpretation of "current", I guess. So yes, change it.